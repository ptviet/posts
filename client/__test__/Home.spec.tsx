import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Home from "../components/Home";

describe("<Home />", () => {
  it("should render and match snapshot", () => {
    const wrapper = shallow(<Home />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
