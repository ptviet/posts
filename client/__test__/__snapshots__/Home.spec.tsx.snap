// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render and match snapshot 1`] = `
<div>
  Hello
</div>
`;
